nucypher/cli/actions.py

```diff
@@ -43,45 +43,40 @@ def load_seednodes(min_stake: int,
     return teacher_nodes
 
 
-def destroy_system_configuration(config_class,
-                                 config_file=None,
-                                 network=None,
-                                 config_root=None,
-                                 force=False,
-                                 log=LOG):
+def destroy_configuration_root(config_root=None, force=False, logs: bool = False) -> str:
+    """CAUTION: This will destroy *all* nucypher configuration files from the filesystem"""
 
     config_root = config_root or DEFAULT_CONFIG_ROOT
 
-    try:
-        character_config = config_class.from_configuration_file(filepath=config_file, domains={network})
+    if not force:
+        message = f"Destroy top-level configuration directory: {config_root}?"
```

Comment:
I would be more forward with what is happening, more like:

"WARNING: this will delete the _entire_ top-level configuration directory. Are you sure you want to do this?"

Reply:
Yeah - good catch, there is actually a whole message written for this right above.  Ill make an update.

Reply:
:heavy_check_mark: